### PR TITLE
CircleCI: Enable circleCI for build test

### PR DESCRIPTION
**Pull request by jeongarmy**
Add config.yml to run circleCI

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            - checkout_code
+      - esp_wrover_kit/hello_with_tash:
+          requires:
+            - checkout_code
+      - imxrt1020-evk/loadable_elf_apps:
+          requires:
+            - checkout_code
+      - rtl8721csm/loadable_apps:
+          requires:
+            - checkout_code
```
Comment (sunghan-chang) [inline .circleci/config.yml:125]: As far as I know, they are all separate jobs. It causes huge docker pull. How about executing build_test only?

**Review by sunghan-chang — COMMENTED** [review]
Is it done for circle ci setting itself?

**Review by jeongchanKim — COMMENTED** [review]
We already have travis ci. 
If we need both, please add comment why we need.
If not, please remove one of them.

**Comment by jeongarmy** [comment]
@jeongchanKim We will replace travisCI to circleCI.

File: .circleci/config.yml (hunk 1 above)
Comment (jeongarmy) [reply .circleci/config.yml]: OK. I agree with your opinion. Please check modified codes.

**Review by an4967 — APPROVED** [review]
LGTM

**Review by jeongchanKim — COMMENTED** [review]
Please split the commit. One for adding circle ci, the other is for removing travis ci.

**Review by an4967 — APPROVED** [review]
LGTM

**Pull request merged**